### ISSUE TITLE
SVG download gone?

### ISSUE THREAD

**Issue by peteruhrig** [int2str/jssyntaxtree#4]
First of all, thanks for the great work, which I have been using and recommending to my students for years!
In phpSyntaxtree there used to be an SVG download, which allowed me to change things in Inkscape when I was unhappy with the original layout (and which was easier to adapt for high-resolution publications). Do you plan to bring this back?

**Comment by int2str** [comment]
Yes, it's gone for now. I didn't think anyone had discovered it or was using SVG download :)
I could bring it back, but it may require some code reorganization. No promises, but I'll think about it.

**Comment by peteruhrig** [comment]
Thank you for your swift reply! Thinking about it, I see no specific need for SVG. Any vector based format - EMF, WMF, PDF with vector content - will allow for high-quality printing. Alternatively, one could think about some high-quality PNG setting with, say, 600dpi, possibly without anti-aliasing.
